src/Sylius/Behat/Page/Admin/Product/CreateConfigurableProductPage.php

```diff
@@ -30,6 +33,13 @@ public function __construct(
         private readonly AutocompleteHelperInterface $autocompleteHelper,
     ) {
         parent::__construct($session, $minkParameters, $router, $routeName);
+
+        $this->defineResourceName();
+    }
+
+    public function defineResourceName(): void
```

Comment:
```suggestion
    public function getResourceName(): void
```
and then in the trait use if instead of `$this->resource`, so the calling the method in the constructor would not be necessary